example.env

```diff
@@ -43,6 +43,10 @@ COMPOSE_PROJECT_NAME=polis-${TAG}
 # Makefile will read this to determine if local services (dynamodb, minio, ollama) should be started.
 LOCAL_SERVICES_DOCKER=true
 
+###### FOR DEV ONLY: DELPHI INSTANCE SIZE ######
+# Uncomment the following *only* if you are running a local dev instance of Delphi.
+# Otherwise, see delphi/DELPHI_AUTOSCALING_SETUP.md for configuring instance size in production.
+# INSTANCE_SIZE=omnipotent
```

Comment:
The default state of example.env should be for developers, such that `cp example.env .env` "just works".

So this should be uncommented. The warnings are not really necessary.
example.env is for sane defaults for local development